src/bin/bat/main.rs

```diff
@@ -101,7 +101,10 @@ pub fn get_languages(config: &Config) -> Result<String> {
                 true
             } else {
                 let test_file = Path::new("test").with_extension(extension);
-                match assets.syntax_for_file_name(test_file, &config.syntax_mapping) {
+                let syntax = assets
+                    .get_syntax_for_file_name(test_file, &config.syntax_mapping)
+                    .unwrap(); // safe since .get_syntaxes() above worked
```

Comment:
definitely out of scope for this PR, but ideally we would have an API which would avoid this. Something like:
```
let syntaxes = assets.get_syntaxes()?;

// and then later:

syntaxes.get_syntax_for_file_name(…) // no unwrap here
```

Reply:
Good idea. Let's try to move in that direction long-term.